README.md

```diff
@@ -34,7 +34,13 @@ Next, enable hot reloading in your webpack config:
     Occurence ensures consistent build hashes, hot module replacement is
     somewhat self-explanatory, no errors is used to handle errors more cleanly.
 
- 3. Add `'webpack-hot-middleware/client'` into the `entry` array.
+ 3. Add `'webpack-hot-middleware/client'` into an array of the `entry` 
+    object. For example:
+    ```js
+    entry: {
+        main: ['./src/main.js', 'webpack-hot-middleware/client']
```

Comment:
Let's put `webpack-hot-middleware/client` as the first item, it doesn't matter, but I recommended it

Reply:
@alexander-akait I made the suggested changes, but I'm getting a permissions error when I try to push the changes. I tried following documentation for personal access tokens, but I'm not sure if I'm doing something wrong or if it's how things are set up on the repo. At work, I just have SSH set up, and I don't contribute much to public repos outside of work.

Reply:
Just use `force` push if you ammend commit

Reply:
I just updated it manually in the browser. Not sure why I was getting errors when trying to push from the CLI...